src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,195 @@
+          h3({ style: styles.headers }, ['Last Updated']),
+          div([Utils.makeStandardDate(snapshot['dct:modified'])])
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Version']),
+          div([_.get('', snapshot)])
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Region']),
+          div([
+            _.map(
+              storage => div({ key: `region-table-${storage.region}` }, [storage.region]),
+              _.uniqBy('region', snapshot.storage || [])
+            )
+          ])
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Contact'])
+        // h(Link, { href: `mailto:fakeemail@fake.org` }, ['fakeemail@fake.org'])
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Data curator'])
+        // div(['Will add later, after data structure is added'])
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Contributors']),
+          _.map(contributor => div({ key: `contributor-list_${contributor}}` }, [contributor]), _.get('contributors', snapshot))
+        ]),
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Cloud provider']),
+          div([
+            _.map(
+              storage => div({ key: `cloud-platform-table-${storage.cloudPlatform}` }, [storage.cloudPlatform]),
+              _.uniqBy('cloudPlatform', snapshot.storage || [])
+            )
+          ])
+        ])
+      ])
+    ])
+  ])
+}
+
+const Sidebar = ({ snapshot, setShowRequestAccessModal }) => {
+  return div({ style: { ...styles.content, width: 300, flexShrink: 0, display: 'flex', flexDirection: 'column', alignItems: 'center' } }, [
+    h2({ className: 'sr-only' }, ['Snapshot Data Details']),
+    div({ style: { backgroundColor: 'white', padding: 20, paddingTop: 0, width: '100%', border: '2px solid #D6D7D7', borderRadius: 5 } }, [
+      div([
+        h3(['Access type']),
+        div([
+          _.get('locked', snapshot) ?
+            h(ButtonSecondary, {
+              style: { fontSize: 16, textTransform: 'none', height: 'unset' },
+              onClick: () => setShowRequestAccessModal(true)
+            }, [
+              div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'center' } }, [
+                icon('lock', { size: 18, style: { marginRight: 10, color: colors.accent() } }),
+                'Request Access'
+              ])
+            ]) :
+            div({ style: { color: colors.primary() } }, [
+              icon('unlock', { size: 18, style: { marginRight: 10 } }),
+              'Open Access'
+            ])
+        ])
+      ]),
+      div([
+        h3({ style: styles.headers }, ['Donor size']),
+        div([_.getOr(0, 'counts.donors', snapshot).toLocaleString()])
+      ]),
```

Comment:
Follow up in a later PR - see if we can clean up the DOM without losing readability